pkg/meta/resources.go

```diff
@@ -26,7 +26,7 @@ var (
 	DefaultSidecarResource = map[corev1.ResourceName]resource.Quantity{
 		corev1.ResourceLimitsCPU:      resource.MustParse("1"),
```

Comment:
Is it 1000 millicores?

Reply:
Yes, this is 1000 millicore. I believe just specifying 1 is equivalent to 1000m.